Visualization/AFW_Display_Demo.ipynb

```diff
@@ -0,0 +1,303 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "afwDisplay.setDefaultBackend('matplotlib')  # Use lsst.afw.display with the matplotlib backend"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now set to display the image. To do this, we:\n",
+    "\n",
+    "* First create a `matplotlib.pyplot` figure using `plt.figure()` -- this will be familiar to anyone with experience using `matplotlib`.\n",
+    "* Then create an alias to the `lsst.afw.display.Display` method that will allow us to display the data to the screen.  This alias will be called `afw_display`.\n",
+    "* Before showing the data on the screen, we have to decide how to apply an image stretch given the data. The algorithm we'll use is `asinh` familiar from SDSS images, with a range of values set by `zscale`. To do this, we use the `scale()` function provided by `lsst.afw.display`. See the `scale()` function definition in the [`interface.py` file of the lsst.afw.display library](https://github.com/lsst/afw/blob/master/python/lsst/afw/display/interface.py).\n",
+    "* Finally, we can display the image. Do do this, we provide the `mtv()` method the `image` member of our calibrated image retrieved by the `butler`. We can then use `plt.show()` to display our figure.\n",
+    "\n",
+    "All these tasks are best done within the same notebook cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()                            #create a matplotlib.pyplot figure\n",
+    "afw_display = afwDisplay.Display()      #get an alias to the lsst.afw.display.Display() method\n",
+    "afw_display.scale('asinh', 'zscale')    #set the image stretch algorithm and range\n",
+    "afw_display.mtv(calexp.image)           #load the image into the display\n",
+    "plt.show()                              #show the corresponding pyplot figure"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Congrats!** We've plotted an image using `lsst.afw.display`!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Step 4) More Information about lsst.afw.display**\n",
+    "\n",
+    "To get some more information about `lsst.afw.display`, we can print the method list to see what's available. The next cell will print `lsst.afw.display` methods to the screen."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "method_list = [func for func in dir(afw_display) if callable(getattr(afw_display, func))]\n",
+    "print(method_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you'd like to learn more about any given function, please see the [`lsst.afw.display` source code](https://github.com/lsst/afw/tree/master/python/lsst/afw/display)."
```

Comment:
We can also recommend the notebook `help()` function for showing the docstrings to the above functions - I'll include this in my PR too.